### DISCUSSION TITLE
How To Continue After Rate Limit

### DISCUSSION THREAD

**Discussion by tinghao-ai** [microsoft/graphrag#399] [Q&A] [answered]
The Error is `0  <graphml xmlns="http://graphml.graphdrawing.or...
❌ create_summarized_entities
None
⠙ GraphRAG Indexer 
├── Loading Input (InputFileType.text) - 113 files loaded (2 filtered) ━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00 0:00:00
├── create_base_text_units
├── create_base_extracted_entities
└── create_summarized_entities
❌ Errors occurred during the pipeline run, see logs for more details.`

because of the OpenAI API Rate Limit. What should I do to continue the previous thread. It's really expensive to start all over again and always have this kind of rate limit porblem.

**Comment by AlonsoGuevara** [comment, marked as answer]
There's a ` --resume` command whe you run an index job, you just need to provide the folder name (the timestamp).
For example ` python -m graphrag.index --root path/to/ --resume 20241506-0123`

This will pick up where your run ended.

**Reply by tinghao-ai** [reply]
Thanks for the answer. But the path should be like this `/Users/Avernus/graphRagTest/output/20240707-023656` or? If I run this command it would just start like empty input output for this folder instead of continue the thread before.
